plugins/module_utils/network/nxos/facts/static_routes/static_routes.py

```diff
@@ -122,9 +122,9 @@ def get_inner_dict(self, conf, inner_dict):
 
         # ethernet1/2/23
         iface = re.match(
-            r".* (Ethernet|loopback|mgmt|port\-channel)(\S*) .*", conf
+            r".* (Ethernet|Vlan|loopback|mgmt|port\-channel)(\S*) .*", conf
```

Comment:
The following change is needed allow Vlan interfaces to show up.
```suggestion
            r".* (Ethernet|Vlan|loopback|mgmt|port\-channel)(.*).*", conf
```